README.md

```diff
@@ -6,70 +6,105 @@
+Our core developers have been building time-series data platforms for years. Based on our best-practices, GreptimeDB is born to give you:
 
-## Quickstart with [GreptimePlay](https://greptime.com/playground)
+* **Easy horizontal scaling**
+
+  Seamless scalability from a standalone binary at edge to a robust, highly available distributed cluster in cloud, with a transparent experience for both developers and administrators.
+
+* **Analyzing time-series data**
+
+  Query your time-series data with SQL and PromQL. Use Python scripts to facilitate complex analytical tasks.
+
+* **Cloud-native distributed database**
+
+  Fully open-source distributed cluster architecture that harnesses the power of cloud-native elastic computing resources.
+
+* **Performance and Cost-effective**
+
+  Flexible indexing capabilities and distributed, parallel-processing query engine, tackling high cardinality issues down. Optimized columnar layout for handling time-series data; compacted, compressed, and stored on various storage backends, particularly cloud object storage with 50x cost efficiency.
+
+* **Compatible with InfluxDB, Prometheus and more protocols**
+
+  Widely adopted database protocols and APIs, including MySQL, PostgreSQL, and Prometheus Remote Storage, etc. [Read more](https://docs.greptime.com/user-guide/clients/overview).
+
+## Try GreptimeDB
+
+### 1. [GreptimePlay](https://greptime.com/playground)
 
 Try out the features of GreptimeDB right from your browser.
 
-## Up & Running
+### 2. [GreptimeCloud](https://console.greptime.cloud/)
 
-The recommended way to install GreptimeDB is via Docker:
+Start instantly with a free cluster.
 
-```shell
-docker pull greptime/greptimedb
-```
+### 3. Docker Image
 
-Start a GreptimeDB container with:
+To install GreptimeDB locally, the recommended way is via Docker:
 
 ```shell
-docker run -p 4000-4003:4000-4003 \
-  -p 4242:4242 -v "$(pwd)/greptimedb:/tmp/greptimedb" \
-  --name greptime --rm \
-  greptime/greptimedb standalone start \
-  --http-addr 0.0.0.0:4000 \
-  --rpc-addr 0.0.0.0:4001 \
-  --mysql-addr 0.0.0.0:4002 \
-  --postgres-addr 0.0.0.0:4003 \
-  --opentsdb-addr 0.0.0.0:4242
+docker pull greptime/greptimedb
 ```
 
-Connect to the server and test:
+Start a GreptimeDB container with:
 
 ```shell
-curl -X POST -d 'sql=SELECT 42&format=csv' http://localhost:4000/v1/sql
+docker run --rm --name greptime --net=host greptime/greptimedb standalone start
 ```
 
-You should get a reply as:
+Read more about [Installation](https://docs.greptime.com/getting-started/installation/overview) on docs.
 
-```
-42
-```
-
-Read more on docs:
+## Getting Started
 
-* [Installation](https://docs.greptime.com/getting-started/installation/overview)
 * [Quickstart](https://docs.greptime.com/getting-started/quick-start/overview)
```

Comment:
Potential - more starting points like [this style](https://github.com/datafuselabs/databend/blob/main/README.md#-getting-started)